ui/src/submissions/index.jsx

```diff
@@ -53,47 +54,54 @@ const JobUpdateSubmissionSuccessPage$ = Loadable({
 class Submissions extends Component {
   render() {
     return (
-      <div className="w-100">
-        <SafeSwitch>
-          <Redirect exact from={SUBMISSIONS} to={SUBMISSIONS_AUTHOR} />
-          <Route
-            exact
-            path={SUBMISSIONS_AUTHOR}
-            component={AuthorSubmissionPage$}
-          />
-          <Route
-            exact
-            path={`${SUBMISSIONS_AUTHOR}/:id`}
-            component={AuthorUpdateSubmissionPage$}
-          />
-          <Redirect
-            exact
-            from={`${SUBMISSIONS_AUTHOR}/:id/success`}
-            to={SUBMISSION_SUCCESS}
-          />
-          <Route
-            exact
-            path={SUBMISSIONS_LITERATURE}
-            component={LiteratureSubmissionPage$}
-          />
-          <Route exact path={SUBMISSIONS_JOB} component={JobSubmissionPage$} />
-          <Route
-            exact
-            path={`${SUBMISSIONS_JOB}/:id`}
-            component={JobUpdateSubmissionPage$}
-          />
-          <Route
-            exact
-            path={`${SUBMISSIONS_JOB}/:id/success`}
-            component={JobUpdateSubmissionSuccessPage$}
-          />
-          <Route
-            exact
-            path={SUBMISSION_SUCCESS}
-            component={SubmissionSuccessPage$}
-          />
-        </SafeSwitch>
-      </div>
+      <>
+        <DocumentHead title="Submit" />
```

Comment:
Why not do ```Literature Submit``` , ```Author Submit``` and so on for each submission page container?

Reply:
as discussed IRL, putting titles for submission pages wasn't a requirement. I just put something for now in order not to display irrelevant titles for some pages. For example `Literature Search` for job submission page, when use moves to job submission after literature search.